registrations/serializers.py

```diff
@@ -339,6 +425,77 @@ class Meta:
         fields = ["id", "email", "language"]
 
 
+class PriceGroupRelatedField(serializers.PrimaryKeyRelatedField):
+    def to_representation(self, value):
+        price_group = PriceGroup.objects.only(
+            "pk", "description_fi", "description_sv", "description_en"
+        ).get(pk=value.pk)
+
+        return {
+            "id": price_group.pk,
+            "description": {
+                lang: getattr(price_group, f"description_{lang}")
+                for lang in ("fi", "sv", "en")
+                if getattr(price_group, f"description_{lang}", None) is not None
+            },
+        }
+
+
+class RegistrationPriceGroupSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(required=False)
+    price_group = PriceGroupRelatedField(queryset=PriceGroup.objects.all())
+    price = serializers.DecimalField(
+        required=False, max_digits=19, decimal_places=2, min_value=0
+    )
+    price_without_vat = serializers.DecimalField(
+        required=False, read_only=True, max_digits=19, decimal_places=2
+    )
+    vat = serializers.DecimalField(
+        required=False, read_only=True, max_digits=19, decimal_places=2
+    )
+    vat_percentage = serializers.DecimalField(
```

Comment:
That's what I though too, but because `ChoiceField` normalizes the value to a string and the API user can provide values like "10.00" or "10.0", the out-of-the-box validation with a `ChoiceField` will fail in those cases and extra code is required to make it work. Thus, I thought this would after all be the simplest solution that I could come up with.